modules/auxiliary/gather/java_rmi_registry.rb

```diff
@@ -6,6 +6,7 @@
 require 'rex/java/serialization'
 
 class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Scanner
```

Comment:
If you're going to make this a scanner, you should move it to `auxiliary/scanner`. Since module deprecation isn't seamless yet (see #12027), I think we can let this one slide for now.

Reply:
Actually, this is a bug.

#9246

Reply:
Ok, good to know! Let me know if there's anything I can do :)

Reply:
Will do! If @busterb or @Green-m don't get to it, I'll see if I can suggest a fix for you to use. :)